en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -989,9 +989,45 @@ class ParentObject extends DataObject
 }
 ```
 
-In this example, when the parent object is deleted, the child specified by the `has_one` relation will also
-be deleted. Note that all relation types (`has_many`, `many_many`, `belongs_many_many`, `belongs_to`, and `has_one`)
-are supported, as are methods that return lists of objects but do not correspond to a physical database relation.
+In this example, when the parent object is deleted, the child specified by the `has_one` relation will also be deleted. The relation types `has_many`, `belongs_to`, and `has_one` are supported, as are methods that return lists of objects but do not correspond to a physical database relation.
+
+### Cascading deletions with `has_many`
+
+When the parent object is deleted, the end child specified by the `cascade_deletes` will also be removed, not unlinked. To remove the link between the two objects, use `has_many` on the relation object directly to create an alternate view of the relationship which can be unlinked.
+
+For example:
+
+```php
+namespace App\Model;
+
+use SilverStripe\ORM\DataObject;
+
+class Team extends DataObject
+{
+    // Define the many-to-many relationship using a custom relation table
+    private static array $many_many = [
+        'Supporters' => [
+            'through' => TeamSupporter::class,
+            'from' => 'Team',
+            'to' => 'Supporter',
+        ],
+    ];
+
+    // Add a different perspective of the relationship data
+    private static array $has_many = [
+        'SupportersRelation' => TeamSupporter::class . '.Team',
+    ];
+
+    // Unlink the relationship when the parent object is deleted
+    private static array $cascade_duplicates = [
+        'SupportersRelation',
+    ];
+
+    // ...
+}
+```
+
+### Cascading deletions with versions
```

Comment:
It may be also worth noting that if you have a a setup when parent model is versioned and the related model is not, the un-publish action will delete the related model because un-publish is a shorthand for delete from live stage. Since the related model is not versioned the deletion occurs and the related model is gone completely.

Reply:
I'd say that is probably out-of-scope for this PR, but there is now a whole sub-section for these 😄